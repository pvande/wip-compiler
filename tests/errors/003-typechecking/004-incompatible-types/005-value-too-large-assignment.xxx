y : byte
y = 256
