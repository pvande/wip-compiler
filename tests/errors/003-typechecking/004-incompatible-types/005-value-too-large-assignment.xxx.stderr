Error: "Cannot assign that argument to that variable; the types don't match"
In [1;37mtests/errors/003-typechecking/004-incompatible-types/005-value-too-large-assignment.xxx[0m on line [1;37m2[0m

> [0;36my = 256[0m
  [0;31m^^^^^^^[0m
